src/components/sendAskMoney/SendAskMoney.vue

```diff
@@ -751,12 +775,18 @@
           }
           if (err.message === "User canceled the dialog box") {
             this.$msg.warning("Transaction en cours annulée")
-            return
+          } else {
+            this.$msg.error(
+              `Une erreur inattendue est survenue pendant le transfert d'argent. ` +
+                `Veuillez nous excuser pour la gêne occasionnée.<br>` +
+                `Vous pouvez réessayer. Si l'erreur persiste, veuillez contacter votre administrateur.`
+            )
```

Comment:
can you throw the error if it is unknown (in addition to inform the user) ?. Otherwise we'll have trouble to debug.

Reply:
@vaab is that OK with you ? If so, can you "resolve conversation" ?